src/Components/CommentSection.jsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import { getArticleComments } from "../utils/articles_api";
+
+export const CommentSection = ({ article_id }) => {
+  const [comments, setComments] = useState([{}]);
+
+  useEffect(() => {
+    getArticleComments(article_id).then((comments) => {
+      setComments(comments);
+    });
+  }, []);
+
+  return (
+    <section className="comment-section">
```

Comment:
It would be useful to add some functionality here to deal with an article that happens to have no comments.

Some isLoading messaging would also be useful here.